lib/u3d_core/command_executor.rb

```diff
@@ -53,41 +53,55 @@ def which(cmd)
       # @param print_command [Boolean] Should we print the command that's being executed
       # @param error [Block] A block that's called if an error occurs
       # @param prefix [Array] An array containg a prefix + block which might get applied to the output
-      # @param loading [String] A loading string that is shown before the first output
       # @param admin [Boolean] Do we need admin privilege for this command?
-      # @param keychain [Boolean] Should we fetch admin rights from the keychain on OSX
       # @return [String] All the output as string
-      def execute(command: nil, print_all: false, print_command: true, error: nil, prefix: nil, loading: nil, admin: false)
+      # @deprecated
+      def execute(command: nil, print_all: false, print_command: true, error: nil, prefix: nil, admin: false)
+        # UI.deprecated("use execute_command instead")
```

Comment:
Is it deprecated or not? If it is, maybe we should indeed log to say so. If not maybe we should remove this comment as it is misleading?

Reply:
I wanted to deprecate it, but we use `execute` in its old form everywhere in the installer.